### PR TITLE
add BGP LARGE_COMMUNITY Path Attribute

### PR DESCRIPTION
**Pull request by Qingtian-Zou**
add support for BGP protocol's LARGE_COMMUNITY Path Attribute, introduced with RFC 8092 and RFC 8195

### CODE REVIEW

**Comment by gpotter2** [comment]
Thanks a lot for the PR. Could you provide a unit test that dissects a real packet ? You can put it insid https://github.com/secdev/scapy/blob/master/test/contrib/bgp.uts

**Comment by Qingtian-Zou** [comment]
Now the dissecting should be OK ...
I think line 2079 to line 2095 in "bgp.py" might be implemented in another way, or it may introduce similar problems for future path attributes.

File: scapy/contrib/bgp.py [hunk 1]
```diff
@@ -2004,6 +2006,31 @@ def post_build(self, p, pay):
...
+# LARGE_COMMUNITY
+#
+
+class BGPPALargeCommunity(Packet):
+    """
+    Provides an implementation of the LARGE_COMMUNITY attribute.
+    References: RFC 8092, RFC 8195
+    """
+
+    class LargeCommunitySegment(Packet):
```
Comment (gpotter2) [inline scapy/contrib/bgp.py:2019]: Could you rename this `BGPLargeCommunitySegment` and move it out of the class?

Comment (Qingtian-Zou) [reply scapy/contrib/bgp.py]: Moved and renamed. I admit I was copying the style of BGPPAASPath class, but now I notice there is another style in "bgp.py".

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**